File: src/core/__tests__/fixtures/sample.ts
import { ReactApp, User } from "@flowconsole/sdk";

const user = new User({
  name: "Alice",
  role: "admin",
  description: "Administrator user",
  tags: ["admin", "user"],
  badge: "gold",
  tone: "muted",
});

const app = new ReactApp({});

user.sendsRequest(app, "Load App");

File: src/core/__tests__/fixtures/sample.py
from flowconsole import User, ReactApp, RestApi

user = User(name='Alice', role='admin', description='Administrator user', tags=['admin', 'user'], badge='gold')
app = ReactApp(name='Dashboard')
api = RestApi(name='Backend API')

user.sendsRequest(app, 'Load App')
app.then(api)
